app/components/Navigation.static.tsx

```diff
@@ -0,0 +1,60 @@
+import { Form } from "react-router";
+import { LogoutType } from "~/routes/action.logout-user";
+
+export function LogoutButton() {
+  return (
+    <Form method="post" action="/action/logout-user">
+      <input type="hidden" name="logoutType" value={LogoutType.ByUser} />
+      <button type="submit" className="kern-link bg-transparent">
+        <span className="kern-icon kern-icon--close kern-icon--default"></span>
+        <span>Abmelden</span>
+      </button>
+    </Form>
+  );
+}
+
+const navigationLinksList = [
+  {
+    name: "Verfahren",
+    iconName: "kern-icon--home",
+    url: "#",
+  },
+  {
+    name: "Mitteilungen",
+    iconName: "kern-icon--mail",
+    url: "#",
+  },
+  {
+    name: "Kalender",
+    iconName: "kern-icon--calendar-today",
+    url: "#",
+  },
+  {
+    name: "Profil",
+    iconName: "kern-icon--easy-language",
+    url: "#",
+  },
+];
+
+export default function Navigation() {
+  return (
+    <nav>
+      <ul className="gap-kern-space-large my-0 items-center justify-between pl-0 sm:flex sm:flex-wrap">
```

Comment:
As mentioned within the `Header`, we could use the mobile layout up to the `xl` breakpoint (therefor only one "hard" layout change would happen for our users for now) and a bit more space has been added to match Figma `gap-kern-space-x-large`:

```suggestion
      <ul className="gap-kern-space-x-large my-0 items-center justify-between pl-0 sm:flex sm:flex-wrap">
```